test/unit/mdc-drawer/mdc-temporary-drawer.test.js

```diff
@@ -322,3 +322,18 @@ test('adapter#isDrawer returns false for a non-drawer element', () => {
   const {root, component} = setupTest();
   assert.isNotOk(component.getDefaultFoundation().adapter_.isDrawer(root));
 });
+
+test('adapter#addBodyClass adds a class to the body, locking the background scroll', () => {
```

Comment:
techincally all `addBodyClass/removeBodyClass` does is add/remove a class to the body (respectively). It's up to the class itself to determine the stylistic behavior. So maybe for these two test descriptions just say `adds a class to [/ removes a class from] the body`.